### PR TITLE
Form branch with 3 ways of forms.

### PR DESCRIPTION
**Pull request by AhmedAmin90**
In this branch, I used 3 ways to build a form in rails that can send parameters to a controller so that it can build an instance of a model with those parameters.
By this form I can create new users and edit these users information.

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="email">User Email</label>
+  <input type="email" id="email" name="user[email]"><br><br>
+   <label for="password">Password</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type="submit" value="Submit">
+</form> 
+-->
+<!--
+<%= form_tag("/users") do %>
```
Comment (descholar-ceo) [inline app/views/users/new.html.erb]: [required] Replace `/users` with `users_path`

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @AhmedAmin90 :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

#### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] To be able to run the project, you can make sure that you change the name of the table from `:user` to `:users` for migrations
- [x] Learn about the inline comments
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by AhmedAmin90** [comment]
@descholar-ceo  Thanks a lot for your positive comments, I did the required changes and I am trying to submit a new code review but the dashboard told me: Project You already have an open code review request for Forms. Is there a new way for code review or there is a problem in the system?

**Comment by descholar-ceo** [comment]
> @descholar-ceo Thanks a lot for your positive comments, I did the required changes and I am trying to submit a new code review but the dashboard told me: Project You already have an open code review request for Forms. Is there a new way for code review or there is a problem in the system?

@AhmedAmin90 Sorry for that inconvenient experience I think now it's fixed

**Comment by mimipeshy** [comment]
### STATUS : CHANGES REQUIRED :red_circle: :recycle: 

Hey there @AhmedAmin90: wave: Great job on working on the feedback given, however, one more change in order to mark this project approved,

- [x] You need to comment out form_tag in views/users/new as from this ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

That's all :smile: , kindly submit another review when done.

_Feel free to reach out to me on slack if you have any questions or something is unclear, I will be glad to help_
Regards
Peris Ndanu

**Comment by AhmedAmin90** [comment]
Thanks, @mimipeshy for your feedback, I changed it, but I was using (if false) way to comment out form_tag as this reference: https://stackoverflow.com/questions/3901619/how-to-comment-lines-in-rails-html-erb-files/13603081

**Comment by mimipeshy** [comment]
### STATUS :APPROVED :partying_face: 

Hey @AhmedAmin90 thanks for the feedback as well, I will take note of it:smile:  This project is therefore approved. Go ahead and merge. All the best and Happy Coding :computer: 

Regards
Peris Ndanu

**Comment by AhmedAmin90** [comment]
Thanks a lot :star: :relaxed: @mimipeshy

**Pull request merged**